sriovnet_test.go

```diff
@@ -117,6 +150,43 @@ func TestGetPfPciFromVfPciError(t *testing.T) {
 	assert.Equal(t, "", pf)
 }
 
+func TestGetPfPciFromAux(t *testing.T) {
+	teardown := setupFakeFs(t)
+	defer teardown()
+	// Create PCI & Auxiliary sysfs layout with FakeFs
+	pfPciAddr := "0000:02:00.0"
+	auxDevName := "mlx5_core.eth.0"
+	auxDevPath := filepath.Join(PciSysDir, pfPciAddr, auxDevName)
+	auxDevLink := filepath.Join(AuxSysDir, auxDevName)
+
+	// PF PCI path and auxiliary device dir
+	_ = utilfs.Fs.MkdirAll(auxDevPath, os.FileMode(0755))
+	_ = utilfs.Fs.MkdirAll(AuxSysDir, os.FileMode(0755))
+	// Auxiliary device link
+	_ = utilfs.Fs.Symlink(auxDevPath, auxDevLink)
+
+	pf, err := GetPfPciFromAux(auxDevName)
+	assert.NoError(t, err)
+	assert.Equal(t, pfPciAddr, pf)
+}
+
+func TestGetPfPciFromAuxError(t *testing.T) {
+	teardown := setupFakeFs(t)
+	defer teardown()
+	// Create PCI sysfs layout with FakeFs
+	pciAddr := "0000:02:00.0"
+	pciPath := filepath.Join(PciSysDir, pciAddr)
+	auxDevName := "mlx5_core.eth.0"
+
+	// PCI path
+	// XXX dlinkin: Why do we need this? We don't check why function had failed anyway
```

Comment:
i dont understand the comment, is it for the `_ = utilfs....` ? if so its for linter errors :)

alternative is to ignore this check for this file.

Reply:
i see so no need add it ? essentially GetPfPciFromAux can fail if it failed to locate aux dev sysfs path or if the path points to a non pci parent device.

perhaps check both ?